Build issue on Fedora rawhide/f39

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Build works on Fedora 37 but fails on 39. 

Sconstruct:

g++ -o build/release/Government.o -c -O2 -flto=auto -ffat-lto-objects -fexceptions -g -grecord-gcc-switches -pipe -Wall -Werror=format-security -Wp,-U_FORTIFY_SOURCE,-D_FORTIFY_SOURCE=3 -Wp,-D_GLIBCXX_ASSERTIONS -specs=/usr/lib/rpm/redhat/redhat-hardened-cc1 -fstack-protector-strong -specs=/usr/lib/rpm/redhat/redhat-annobin-cc1 -m64 -mtune=generic -fasynchronous-unwind-tables -fstack-clash-protection -fcf-protection -fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -std=c++11 source/Government.cpp
source/GameWindow.cpp: In static member function ‘static bool GameWindow::Init()’:
source/GameWindow.cpp:220:35: error: ‘VSyncState’ is not a member of ‘Preferences’
  220 |         if(!SetVSync(Preferences::VSyncState()))
      |                                   ^~~~~~~~~~
source/GameWindow.cpp: At global scope:
source/GameWindow.cpp:329:6: error: ‘bool GameWindow::SetVSync’ is not a static data member of ‘class GameWindow’
  329 | bool GameWindow::SetVSync(Preferences::VSync state)
      |      ^~~~~~~~~~
source/GameWindow.cpp:329:40: error: ‘VSync’ is not a member of ‘Preferences’
  329 | bool GameWindow::SetVSync(Preferences::VSync state)
      |                                        ^~~~~
scons: *** [build/release/GameWindow.o] Error 1
scons: building terminated because of errors.

CMake:
/home/builduser/rpmbuild/BUILD/endless-sky-0.10.0/source/GameWindow.cpp:220:35: error: 'VSyncState' is not a member of 'Preferences'
  220 |         if(!SetVSync(Preferences::VSyncState()))
      |                                   ^~~~~~~~~~
/home/builduser/rpmbuild/BUILD/endless-sky-0.10.0/source/GameWindow.cpp: At global scope:
/home/builduser/rpmbuild/BUILD/endless-sky-0.10.0/source/GameWindow.cpp:329:6: error: 'bool GameWindow::SetVSync' is not a static data member of 'class GameWindow'
  329 | bool GameWindow::SetVSync(Preferences::VSync state)
      |      ^~~~~~~~~~
/home/builduser/rpmbuild/BUILD/endless-sky-0.10.0/source/GameWindow.cpp:329:40: error: 'VSync' is not a member of 'Preferences'
  329 | bool GameWindow::SetVSync(Preferences::VSync state)
      |                                        ^~~~~
gmake[2]: *** [source/CMakeFiles/EndlessSkyLib.dir/build.make:821: source/CMakeFiles/EndlessSkyLib.dir/GameWindow.cpp.o] Error 1
gmake[2]: *** Waiting for unfinished jobs....
gmake[2]: Leaving directory '/home/builduser/rpmbuild/BUILD/endless-sky-0.10.0/redhat-linux-build'
gmake[1]: *** [CMakeFiles/Makefile2:905: source/CMakeFiles/EndlessSkyLib.dir/all] Error 2
gmake[1]: Leaving directory '/home/builduser/rpmbuild/BUILD/endless-sky-0.10.0/redhat-linux-build'
gmake: *** [Makefile:149: all] Error 2
error: Bad exit status from /var/tmp/rpm-tmp.Y9vnXi (%build)




### Steps to Reproduce

Update Fedora srpm to 0.10.0.
attempt Rawhide build.

### Expected Behavior

Build succeeds.

### Screenshots

_No response_

### Link to save file

_No response_

### Operating System

Fedora Linux 39 x86_64 

### Game Source

GitHub Releases

### Game Version

0.10.0

### Additional Information

I'm one of the Fedora RPM maintainers.

--- a/source/Preferences.h
+++ b/source/Preferences.h
@@ -16,6 +16,7 @@ this program. If not, see <https://www.gnu.org/licenses/>.
 #ifndef PREFERENCES_H_
 #define PREFERENCES_H_
 
+#include <cstdint>
 #include <string>
 
 

